packages/contracts/contracts/L1/Colosseum.sol

```diff
@@ -401,7 +645,7 @@ contract Colosseum is Initializable, Semver {
 
             // If the challenger doesn't close a challenge even though the asserter has timed out,
             // the challenger is considered to have timed out.
-            if (_isPast(_challenge.timeoutAt + CHALLENGE_TIMEOUT)) {
+            if (_isPast(_challenge.timeoutAt + BISECTION_TIMEOUT)) {
```

Comment:
According to the comment above, I think `BISECTION_TIMEOUT` is intended by him.
But as suggested by @seolaoh, we also need to cover the case when challenger timed out in proving.
This is my suggestion:

```go
// ...
        if (!_isAbleToBisect(_challenge.seqSize, _challenge.turn)) {
            if (isPast(_challenge.timeoutAt)) {
                return ChallengeStatus.CHALLENGER_TIMEOUT;
            }
            return ChallengeStatus.READY_TO_PROVE;
        }

        // Check if it's a timed out challenge.
        if (_isPast(_challenge.timeoutAt)) {
            // timeout on challenger turn
            if (isChallengerTurn) {
                return ChallengeStatus.CHALLENGER_TIMEOUT;
            }

            // If the challenger doesn't close a challenge even though the asserter has timed out,
            // the challenger is considered to have timed out.
            if (_isPast(_challenge.timeoutAt + BISECTION_TIMEOUT)) {
                return ChallengeStatus.CHALLENGER_TIMEOUT;
            }

            // timeout on asserter turn
            return ChallengeStatus.ASSERTER_TIMEOUT;
        }
// ...
```

Reply:
Ah, I was confused. Please ignore my comment.
It should be `PROVING_TIMEOUT` since when asserter timed out, challenger should prove in `PROVING_TIMEOUT`. Or, it is `CHALLENGER_TIMEOUT`.
Accordingly, the comment should also be modified.

Reply:
I will fix the comment to be a little more accurate.
```
// If the asserter times out and the challenger does not prove fault,
// the challenger is assumed to have timed out.
```